### PR TITLE
BISERVER-13014 "Allow scheduling" and "Hidden" checkboxes are not imported properly

### PR DESCRIPTION
**Pull request by AliaksandrDrebenchuk**
checkboxes was fixed

### CODE REVIEW

**Comment by AliaksandrDrebenchuk** [comment]
@rfellows @pamval @rmansoor @pentaho-nbaker please take a look PR

**Comment by AliaksandrDrebenchuk** [comment]
@rfellows @pamval @rmansoor @pentaho-nbaker please take a look PR

**Comment by AliaksandrDrebenchuk** [comment]
This PR fixes problems of https://github.com/pentaho/pentaho-platform/pull/2990 which was reverted

File: extensions/test-src/org/pentaho/platform/plugin/services/importexport/exportManifest/bindings/EntityMetaDataTest.java [hunk 1]
```diff
@@ -1,33 +1,30 @@
 /*
- * ******************************************************************************
+ * This program is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU General Public License, version 2 as published by the Free Software
+ * Foundation.
  *
- * Copyright (C) 2002-2015 by Pentaho : http://www.pentaho.com
+ * You should have received a copy of the GNU General Public License along with this
+ * program; if not, you can obtain a copy at http://www.gnu.org/licenses/gpl-2.0.html
```
Comment (rmansoor) [inline extensions/test-src/org/pentaho/platform/plugin/services/importexport/exportManifest/bindings/EntityMetaDataTest.java]: @AliaksandrDrebenchuk please use the proper license header for extensions project

File: extensions/test-src/org/pentaho/platform/plugin/services/importer/MetadataImportHandlerIT.java [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
+/*
  * This program is free software; you can redistribute it and/or modify it under the
- * terms of the GNU Lesser General Public License, version 2.1 as published by the Free Software
+ * terms of the GNU General Public License, version 2 as published by the Free Software
  * Foundation.
  *
- * You should have received a copy of the GNU Lesser General Public License along with this
- * program; if not, you can obtain a copy at http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html
+ * You should have received a copy of the GNU General Public License along with this
+ * program; if not, you can obtain a copy at http://www.gnu.org/licenses/gpl-2.0.html
```
Comment (rmansoor) [inline extensions/test-src/org/pentaho/platform/plugin/services/importer/MetadataImportHandlerIT.java]: @AliaksandrDrebenchuk  please use the proper license header for extensions project

File: extensions/test-src/org/pentaho/platform/plugin/services/importer/SolutionImportHandlerNamingIT.java [hunk 3]
```diff
@@ -1,25 +1,22 @@
...
+/*
  * This program is free software; you can redistribute it and/or modify it under the
- * terms of the GNU Lesser General Public License, version 2.1 as published by the Free Software
+ * terms of the GNU General Public License, version 2 as published by the Free Software
  * Foundation.
  *
- * You should have received a copy of the GNU Lesser General Public License along with this
- * program; if not, you can obtain a copy at http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html
+ * You should have received a copy of the GNU General Public License along with this
+ * program; if not, you can obtain a copy at http://www.gnu.org/licenses/gpl-2.0.html
```
Comment (rmansoor) [inline extensions/test-src/org/pentaho/platform/plugin/services/importer/SolutionImportHandlerNamingIT.java]: @AliaksandrDrebenchuk please use a proper license header for extensions project

**Comment by AliaksandrDrebenchuk** [comment]
@rmansoor As for manual test, please, look at my comment in the http://jira.pentaho.com/browse/BISERVER-13014. "The issue steps can be reproduced when ImportHandler is ignore the values from exportManifest.xml file. In this case please try to use another extension or re-configurate the ImportHandlerMimeTypeDefinitions.xml"

**Comment by rmansoor** [comment]
@AliaksandrDrebenchuk why is the values from exportManifest being ignored

**Comment by AliaksandrDrebenchuk** [comment]
@rmansoor Please look at the https://github.com/pentaho/pentaho-platform/blob/d342d38bc1d2e9506884db671eec7c80a2751f87/extensions/src/org/pentaho/platform/plugin/services/importer/PRPTImportHandler.java. It's a special logic for report files and I have no idea why exist handler which ignores the "hidden" value and sets the new.

**Comment by AliaksandrDrebenchuk** [comment]
@rfellows @pamval @rmansoor @pentaho-nbaker please review

**Comment by rmansoor** [comment]
@AliaksandrDrebenchuk I tried using other content type but the flag set in the manifest does no take effect. I also went to advanced import option and select to take value from the imported file and that did not help either.

**Comment by rmansoor** [comment]
@AliaksandrDrebenchuk this work seems incomplete. It is too risky to fix this right now for 7.0. I am closing this PR and we will continue to work on it post 7.0

**Pull request closed without merging**